dotnet/src/Connectors/Connectors.AzureOpenAI/Core/AzureClientCore.ChatCompletion.cs

```diff
@@ -20,6 +21,19 @@ namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
 /// </summary>
 internal partial class AzureClientCore
 {
+#pragma warning disable IDE1006 // Naming Styles
+#pragma warning disable AOAI001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+    // These delegate instances are used to call the internal overloads of CreateResponseAsync and CreateResponseStreamingAsync that accept
+    // a RequestOptions. These should be replaced once a better way to pass RequestOptions is available.
```

Comment:
@rogerbarreto, this comment looks like a copy/paste error.

This use of reflection is also highly undesirable. Seems like the removal of this without a replacement may have been an oversight (I've asked), or maybe Patch is somehow supposed to be used to achieve the same thing. It'd be good to undo this as soon as there's something better.